packages/drivers/odsp-driver/src/odspError.ts

```diff
@@ -27,17 +27,19 @@ export function throwOdspNetworkError(
     errorMessage: string,
     statusCode: number,
     response?: Response,
+    responseText?: string,
 ): never {
     const claims = statusCode === 401 && response?.headers ? parseAuthErrorClaims(response.headers) : undefined;
 
     const networkError = createOdspNetworkError(
-        response ? `${errorMessage} (${response.statusText})` : errorMessage,
+        response && response.statusText !== "" ? `${errorMessage} (${response.statusText})` : errorMessage,
         statusCode,
         response ? numberFromHeader(response.headers.get("retry-after")) : undefined, // seconds
         claims);
 
     const errorAsAny = networkError as any;
 
+    errorAsAny.response = responseText;
```

Comment:
Should we name it as responseText just to be explicit?

Reply:
I think "text" is just the representation (i.e. it can be binary array, stream, etc.) that makes sense in-memory. It does not feel like it adds a value when looking in Kusto, as everything is pretty much either text or some simple primitive (i.e. number) there and we do not differentiate type based on name of property